business/src/main/java/in/koreatech/business/feature/insertstore/insertdetailinfo/InsertDetailInfoScreen.kt

```diff
@@ -0,0 +1,44 @@
+package `in`.koreatech.business.feature.insertstore.insertdetailinfo
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
+import `in`.koreatech.koin.domain.model.owner.insertstore.StoreBasicInfo
+import org.orbitmvi.orbit.compose.collectAsState
+
+@Composable
+fun InsertDetailInfoScreenImpl(
+    modifier: Modifier = Modifier,
+    onBackPress: () -> Unit,
+    stroeBasicInfo: StoreBasicInfo = StoreBasicInfo(),
+    viewModel: InsertDetailInfoScreenViewModel = hiltViewModel()
+) {
+    val state = viewModel.collectAsState().value
+    InsertDetailInfoScreen(
+        storeNumber = state.storeAddress
+    )
+}
+
+
+@Composable
+fun InsertDetailInfoScreen(
+    modifier: Modifier = Modifier,
+    storeNumber: String = "010-1111-1111"
```

Comment:
가게 번호가 잘못 내려올수도 있기 떄문에 빈 값으로 두는게 좋아보입니다.
```suggestion
    storeNumber: String = ""
```

Reply:
넵 알겠습니다!